src/internet_identity/src/storage.rs

```diff
@@ -289,9 +297,15 @@ pub struct Storage<M: Memory> {
     lookup_anchor_with_device_credential_memory_wrapper: MemoryWrapper<ManagedMemory<M>>,
     lookup_anchor_with_device_credential_memory:
         StableBTreeMap<StorableCredentialId, StorableAnchorNumber, ManagedMemory<M>>,
+
     lookup_application_with_origin_memory_wrapper: MemoryWrapper<ManagedMemory<M>>,
-    lookup_application_with_origin_memory:
+
+    // This field is deprecated in favor of `lookup_application_with_origin_memory`.
+    // TODO[ID-354]: Remove this after the production data is migrated to the new map.
+    lookup_application_with_origin_memory_old:
```

Comment:
Can we just change the name like this and be backwards compatible? Do I understand correctly, that this will read the part of the current memory, but it will be set in another field?

Reply:
+1